frontend/src/scenes/plugins/plugin/PluginCard.tsx

```diff
@@ -133,11 +134,9 @@ export function PluginCard({
                                 <Tag color="blue">
                                     <LoadingOutlined /> Checking for updates…
                                 </Tag>
-                            ) : latestTag && tag !== latestTag ? (
-                                <Tag color="volcano">
-                                    <CloudDownloadOutlined /> Update available!
-                                </Tag>
-                            ) : latestTag && tag === latestTag ? (
+                            ) : url && latestTag && tag && tag !== latestTag ? (
+                                <UpdateAvailable url={url} tag={tag} latestTag={latestTag} />
+                            ) : url && latestTag && tag && tag === latestTag ? (
```

Comment:
This looks a bit weird with three conditions duplicated, could they be extracted somehow?